server/app/services/question/types/QuestionDefinition.java

```diff
@@ -54,21 +80,28 @@ public String serializeAsString() {
     }
   }
 
+  // Note: All the methods below are @JsonIgnore-d because they all pull information from the {@code
```

Comment:
Nit: looks like only methods that take no params need to be @JsonIgnored - would it be clearer to specify that in this comment? It took me a minute to figure out why a couple methods aren't ignored. I suspect I'm not the only one who hasn't had to think much about how the Jackson serialization annotations work :)

Reply:
great question + eagle eyes! i updated the comment to explain why we don't need the ignore annotation on those methods.